generators/src/controller/index.d.ts

```
@@ -0,0 +1,25 @@
+    get initializing(): {
+        validateEntityName: () => void;
+    };
+    configuring(): void;
+    writing(): void;
+    end(): void;
+    _generateAppCode(configOptions: ConfigOptions): void;
+    _generateDbMigrationConfig(configOptions: ConfigOptions): void;
+    _generateFlywayMigration(configOptions: ConfigOptions): void;
+    _generateLiquibaseMigration(configOptions: ConfigOptions): void;
+}
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)